### PR TITLE
Build for arches; add run_exports

### PR DESCRIPTION
**Pull request by xhochy**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by xhochy** [comment]
There isn't more history than 0.9.x on https://abi-laboratory.pro/index.php?view=timeline&l=lmdb thus I opted for a "x.x" run_exports.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,8 +11,10 @@ source:
     - patches/Makefile.patch
 
 build:
-  number: 0
+  number: 1
   skip: true  # [win]
+  run_exports:
+    - {{ pin_subpackage("lmdb", max_pin="x.x") }}
```
Comment (h-vetinari) [inline recipe/meta.yaml]: I'm not even sure it's that. I'd go with patch-level (or exact pins) unless we have good evidence to the contrary.

**Review by h-vetinari — COMMENTED** [review]
I guess we can "tell" the migrators about this retroactively.

File: recipe/meta.yaml (hunk 1 above)
Comment (xhochy) [reply recipe/meta.yaml]: ABI laboratory is normally enough evidence. lmdb doesn't look like it will be getting more releases soon.

Comment (h-vetinari) [reply recipe/meta.yaml]: It's missing the last 5 versions, but if you feel confident that lmdb didn't break ABI in the meantime - fine by me.

**Review by h-vetinari — APPROVED** [review]

**Comment by xhochy** [comment]
@h-vetinari So, I pinned to `patch` now but you accepted the PR before that. What should we do now? :D

**Comment by h-vetinari** [comment]
I'm fine either way though I think we can easily merge as-is. It should be a strict improvement to what we had before (where this feedstock basically wasn't used at all). We can still loosen it later - easier than marking stuff as broken. Otherwise just revert or force-push out the last commit 🙃

**Pull request merged**